### PR TITLE
Update RLC example

### PR DESCRIPTION
**Pull request by eggebee6**
Contains a few small updates to the example model and to the data logging/capture features

### CODE REVIEW

File: demo_scenario.py [hunk 1]
```diff
@@ -31,22 +31,26 @@ def set_up_scenario(
...
             digital_signals = [])
 
+        CAPTURE_DURATION = 0.1
+
         # Create and schedule switch close and open events
         close_event = typhoon_automator.Utility.create_callback_event(
             message = "Closing switch",
             callback = DemoScenario.close_switch)
-        close_time = random.uniform(0.0, self._duration / 2)
+        close_time = random.uniform(CAPTURE_DURATION / 2, self._duration / 2)
```
Comment (soniakbendre) [inline demo_scenario.py:40]: this makes it so that at least some of the scenario plays out, right?

Comment (soniakbendre) [reply demo_scenario.py:40]: before the close event**

File: demo_scenario.py [hunk 2]
```diff
@@ -31,22 +31,26 @@ def set_up_scenario(
                 "V_cap"],
             digital_signals = [])
 
+        CAPTURE_DURATION = 0.1
```
Comment (soniakbendre) [inline demo_scenario.py:34]: is this used as an offset?

**Comment by soniakbendre** [comment]
looks good!

**Comment by soniakbendre** [comment]
are you able to merge this? i might have pressed a wrong button

File: demo_scenario.py (hunk 1 above)
Comment (eggebee6) [reply demo_scenario.py:40]: There was a problem with random values scheduling things at invalid times.  I forget the exact reason for delaying the close time, but we'll leave this as-is for now (the example will be updated later)

File: demo_scenario.py (hunk 2 above)
Comment (eggebee6) [reply demo_scenario.py:34]: This specifies the time duration of the high resolution data capture (i.e. recording the data on each simulation time step, which will rapidly generate large amounts of data and hence the brief capture window)

**Pull request merged**